spec/features/user_cannot_access_admin_privileges_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+
+RSpec.describe 'Access', type: :feature do
+  create_user
+    feature 'user' do
+      it 'cannot access admin screens' do
+
+      visit root_path
+      click_link "Login"
+
+      fill_in "Username", with: user_one.username
+      fill_in "Password", with: 'password'
+
+      click_button "Login"
+
+      visit new_admin_category_path
+
+      expect(page).to have_content("404 Page Not Found")
+    end
```

Comment:
`end` at 20, 4 is not aligned with `it 'cannot access admin screens' do` at 7, 6.